site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -8,24 +8,25 @@ description: How to respond to browser triggers (events) from a Chrome Extension
 
 Extensions are event-based programs used to modify or enhance the Chrome browsing experience. Events
 are browser triggers, such as navigating to a new page, removing a bookmark, or closing a tab.
-Extensions monitor these events using scripts in their background
-[service worker](/docs/extensions/mv3/migrating_to_service_workers), which then react
-with specified instructions.
+Extensions monitor these events using scripts in their background [service
+worker][doc-sw-migration], which then react with specified instructions.
 
-A background service worker is loaded when it is needed, and unloaded when it goes idle. Some examples of
-events include:
+A background service worker is loaded when it is needed, and unloaded when it goes idle. Some
+examples include:
 
 - The extension is first installed or updated to a new version.
 - The background page was listening for an event, and the event is dispatched.
 - A content script or other extension [sends a message.][1]
 - Another view in the extension, such as a popup, calls [`runtime.getBackgroundPage`][2].
 
-Once it has been loaded, a service worker keeps running as long as it is performing an action,
-such as calling a Chrome API or issuing a network request. Additionally, the service worker won't
-unload until all visible views and all message ports are closed.
+Once it has been loaded, an extension's service worker generally keeps running as long as it is
+performing an action, such as calling a Chrome API or issuing a network request. 
 
 {% Aside %}
-Opening a view doesn't cause the service worker to load, but only prevents it from closing once loaded.
+
+Opening a view doesn't cause the service worker to load, but only prevents it from closing once
```

Comment:
This was copied from MV2 docs too, I guess, because in MV3 the service worker unloads as usual regardless of an open popup/tab, i.e. 30 seconds by default or 5 minutes + 30 seconds when there's a chrome.runtime port.